composer.json

```diff
@@ -125,7 +125,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.5.x-dev"
```

Comment:
This must be reverted

Reply:
Done - I had modified the current master, but switched to the 2.5 branch

Reply:
done